arex-instrumentation/servlet/arex-httpservlet/src/main/java/io/arex/inst/httpservlet/ServletExtractor.java

```diff
@@ -111,6 +111,10 @@ private Map<String, String> getResponseHeaders() {
     }
 
     private String getRequest() {
+        String contentType = adapter.getContentType(httpServletRequest);
+        if (StringUtil.isNotEmpty(contentType) &&  contentType.contains("application/json")){
```

Comment:
Please consider content-type as ·application/xml· or other custom message converters.

Reply:
sure，I will consider other types and expand on them